### PR TITLE
Feature/oruk error banner

### PR DESCRIPTION
**Pull request by h0usemaintenance**

### CODE REVIEW

File: src/components/banner/ErrorFallback.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  return (
+    <main className="main-container">
+      <div className="error-page-container">
+        <h1>Something went wrong</h1>
+        <p role="alert" className="error-detail">Unable to load the requested page click to try again. 
+        If problem persists please contact the site administrator at <a href="mailto:hello@openreferraluk.org">hello@openreferraluk.org</a></p>
+        <a href="/" className="button button-secondary button-header">Open Referral UK Home</a>
+      </div>
+    </main>
+  );
```
Comment (yemijaiyeola) [inline src/components/banner/ErrorFallback.js:11]: Cool.  We are not using error prop at the moment.  But will be used server side to write error to log.

File: src/App.js [hunk 2]
```diff
@@ -125,14 +121,13 @@ function App() {
...
                 // set data from strapi to state vars
-                console.log("homeprops", data)
                 setHomeProps((data));
                 // setBodyText(data.projectOverview);
                 if (data.top_menu) {
                     setTopMenuId(data.top_menu.id);
                 }
 
-            }).catch(err => console.log("do something with error as required"));
+            }).catch(err => console.log("An exception occurred ", err));
```
Comment (yemijaiyeola) [inline src/App.js]: we don't want to write "err" to the browser console here.  we use err server side to log to file

**Review by yemijaiyeola — CHANGES_REQUESTED** [review]
Contentconsumer and datafetch, I think still logging to console
Screenshot 2021-08-18 at 16 59 31

File: src/components/banner/ErrorFallback.js (hunk 1 above)
Comment (h0usemaintenance) [reply src/components/banner/ErrorFallback.js:11]: Yes, i left it unused following our discussion a few days ago.

File: src/App.js (hunk 2 above)
Comment (h0usemaintenance) [reply src/App.js]: The error has been returned instead of logging it in App.js. The contentConsumer and datafetch files have been cleaned as well.

Comment (h0usemaintenance) [reply src/App.js]: The return statements added above have been reverted to console logs following our discussion earlier as there is currently no process to consume/ action the error(s) returned.

**Review by yemijaiyeola — APPROVED** [review]

**Pull request merged**